lib/spack/spack/compilers/intel.py

```diff
@@ -65,6 +65,26 @@ def cxx14_flag(self):
         else:
             return "-std=c++14"
 
+    @property
+    def c99_flag(self):
+        if self.version < ver('12'):
+            raise UnsupportedCompilerFlag(self,
+                                          "the C99 standard",
+                                          "c99_flag",
+                                          "< 12")
+        else:
+            return "-std=c99"
+
+    @property
+    def c11_flag(self):
+        if self.version < ver('16'):
+            raise UnsupportedCompilerFlag(self,
+                                          "the C11 standard",
+                                          "c11_flag",
+                                          "< 16")
+        else:
+            return "-std=c1x"
+
```

Comment:
Should be:
| Ver >= | C99 | C11 |
| - | - | - |
| 12 | -std=c99 | X |
| 15 | -std=c99 | -std=c11 |

Reply:
I originally used version 16 here because many of the [language features](https://software.intel.com/en-us/articles/c11-support-in-intel-c-compiler) were not completely implemented until then.